etc/kayobe/environments/ci-multinode/README.md

```diff
@@ -77,17 +77,19 @@ cd ../stackhpc-kayobe-config
 source kayobe-env --environment ci-multinode
 ```
 
-6. Add hooks for `configure-vxlan.yml` and `growroot.yml`
 
+6. Add hooks for `grow-root.yml`, `fix-hostname.yml`, `fix-networking.yml`, `configure-vxlan.yml`
 ```
 mkdir -p ${KAYOBE_CONFIG_PATH}/hooks/overcloud-host-configure/pre.d
+mkdir -p ${KAYOBE_CONFIG_PATH}/hooks/seed-host-configure/pre.d
 cd ${KAYOBE_CONFIG_PATH}/hooks/overcloud-host-configure/pre.d
-ln -s ${KAYOBE_CONFIG_PATH}/ansible/growroot.yml 40-growroot.yml
-```
-```
-mkdir -p ${KAYOBE_CONFIG_PATH}/hooks/overcloud-host-configure/post.d
-cd ${KAYOBE_CONFIG_PATH}/hooks/overcloud-host-configure/post.d
-ln -s ${KAYOBE_CONFIG_PATH}/ansible/configure-vxlan.yml 50-configure-vxlan.yml
+ln -s ${KAYOBE_CONFIG_PATH}/ansible/growroot.yml 5-growroot.yml
+ln -s ${KAYOBE_CONFIG_PATH}/ansible/fix-hostname.yml 10-fix-hostname.yml
+ln -s ${KAYOBE_CONFIG_PATH}/ansible/fix-networking.yml 15-fix-networking.yml
+ln -s ${KAYOBE_CONFIG_PATH}/ansible/configure-vxlan.yml 20-configure-vxlan.yml
+cd ${KAYOBE_CONFIG_PATH}/hooks/seed-host-configure/pre.d
+ln -s ${KAYOBE_CONFIG_PATH}/ansible/growroot.yml 5-growroot.yml
```

Comment:
Surprised that the seed doesn't need fix-networking.yml

Reply:
Good spot. Missed the symlink when writing the docs. Added fix-networking to seed-host-configure.